### DISCUSSION TITLE
developing a portable, solar powered  modem for LowFER

### DISCUSSION THREAD

**Discussion by hatonthecat** [markqvist/Reticulum#171] [Ideas]
Hi! 

I recently read an IEEE article on the HamMessenger, https://spectrum.ieee.org/ham-radio-text-hacking, and I became interested in modems for even lower power:

 http://www.ka7oei.com/qrss1.html 
It states, "Communications theory (Shannon's Law) states that if you were willing to transmit your data infinitely slowly you could communicate with infinitely narrow detection bandwidth and infinitely low (not zero) power.  It should go without saying that there are practical limits to how slow you would go to convey useful information in a reasonable amount of time."

The Arduino in the OpenModem uses an ATmega1284p . While it uses a relatively low power consumption: "~31mA at 5V", I am curious if this microcontroller could be used to run the entire HAM messenger on solar: https://ambiq.com/apollo4-blue-plus/ Other projects have used the MCU: https://github.com/tudssl/engage )it runs b/t 4uA/mhz for Apollo4 to 6uA/mhz for Apollo3.

For LowFER, it runs on the unlicensed band of 136-138kHz. While LoRA can run at 400- 800 Mhz, even as low as 11bps:

" LoRa can be configured to use bandwidths in pre-determined steps from 7.8 KHz to 500 KHz in the sub-gigahertz bands, and from 250 KHz to 1.6 MHz in the 2.4 GHz band. Generally speaking, choosing a narrower bandwidth will result in a slower transfer rate, but improved range. Choosing a wider bandwidth will result in an increased data rate, but a shorter range." https://unsigned.io/articles/2022_05_05_understanding-lora-parameters.html

I'd like to start out brainstorming a proof-of-concept LowFER or MedFER Messenger by porting Windows based DSP Spectran or Argo software (or another version, if it exists) to Cortex-M based MCUs: to decode ~90 second dits (not for waterfall visualization but for demodulation and conversion into text):
https://www.qsl.net/m0ayf/What-is-QRSS.html : 
"QRSS generally means that the CW sending speed is below 2-3 WPM - usually much slower than that.  Let's take as a rather extreme example, the VA3LK beacon on 137.79 Hz.  This experimental beacon has operated at a "dit" rate of one dit every ninety seconds - that's about .0133 words-per-minute, or about 0.8 words-per-hour.  This also implies that the detection bandwidth for such a signal (see the equations [below](http://www.ka7oei.com/qrss1.html#bw_speed)) should be at least 0.033 Hz - that's 33 Millihertz (mHz - notice the small "m"!)  Going much much narrower than this and the "dits" will start to run together and sacrifice "intelligibility.

Compare this to the 2.4 Hz bandwidth for a moment:  33 mHz (notice the small "m") is 72000 times narrower than an SSB bandwidth - a difference of over 48 db.  A better example would be to compare this with a 30 Hz CW filter - a "reasonable" value for a well-designed CW filter.  This would still be 900 times narrower than that - over 29 db difference in the receive signal/noise ratio.  (Once again, we are ignoring psychoacoustics.)"

Then integrating on a Single board computer, a batteryless-centered design- i.e supercaps or lithium ion capacitors:
https://www.youtube.com/watch?v=LDzxifG7VeA&t=237s <-(similar to the batteryless bluetooth beacon, but on ham bands)
https://www.tindie.com/products/jaspersikken/solar-harvesting-into-lithium-ion-capacitor/ :
"If your application draws 30mA from the 3.3V output, then it will run for: 250F(3.78V-2.49V)90%/0.03A= 10750s = 2.7 hrs"

In the U.S., the maximum transmit power for MedFER with a 3 meter antenna is 100mW:
https://lwca.net/sitepage/part15/index.htm
http://www.ka7oei.com/psk_medfer.html
 A portable phone wouldn't be able to use 3 meter antenna- maybe only 7 centimeters or 30cm thus lower range would be expected, but the idea here is to explore bands other than LoRa to transmit even a few characters all on solar power (and initially, without GPS, since today's tech probably can't run it without larger solar panels in real time). 

So theoretically, one could have a portable, fully solar powered phone transmitting a signal for 2.7hrs running at 100mW to send a few bytes of data (the microcontroller alone uses less than 1mA: 5uAx 96Mhz=960uA)

While I understand rechargeable batteries are not super expensive, the idea is that by using lithium ion capacitors, or supercaps, it would create a positive-feedback loop of developing slower transmission methods for lower priority projects (as an amateur project).  Thanks for reading!

**Comment by stevewa2066** [comment]
Could even do a digital signage type application. Doesn’t matter if it
takes a few try’s to get the data through .

Steve  N0FPF

On Sat, Dec 3, 2022 at 9:45 AM hatonthecat ***@***.***> wrote:

> Hi!
>
> I recently read an IEEE article on the HamMessenger,
> https://spectrum.ieee.org/ham-radio-text-hacking, and I became interested
> in modems for even lower power:
>
> http://www.ka7oei.com/qrss1.html
> It states, "Communications theory (Shannon's Law) states that if you were
> willing to transmit your data infinitely slowly you could communicate with
> infinitely narrow detection bandwidth and infinitely low (not zero) power.
> It should go without saying that there are practical limits to how slow you
> would go to convey useful information in a reasonable amount of time."
>
> The Arduino in the OpenModem uses an ATmega1284p . While it uses a
> relatively low power consumption: "~31mA at 5V", I am curious if this
> microcontroller could be used to run the entire HAM messenger on solar:
> https://ambiq.com/apollo4-blue-plus/ Other projects have used the MCU:
> https://github.com/tudssl/engage )it runs b/t 4uA/mhz for Apollo4 to
> 6uA/mhz for Apollo3.
>
> For LowFER, it runs on the unlicensed band of 136-138kHz. While LoRA can
> run at 400- 800 Mhz, even as low as 11bps:
>
> " LoRa can be configured to use bandwidths in pre-determined steps from
> 7.8 KHz to 500 KHz in the sub-gigahertz bands, and from 250 KHz to 1.6 MHz
> in the 2.4 GHz band. Generally speaking, choosing a narrower bandwidth will
> result in a slower transfer rate, but improved range. Choosing a wider
> bandwidth will result in an increased data rate, but a shorter range."
> https://unsigned.io/articles/2022_05_05_understanding-lora-parameters.html
>
> I'd like to start out brainstorming a proof-of-concept LowFER or MedFER
> Messenger by porting Windows based DSP Spectran or Argo software (or
> another version, if it exists) to Cortex-M based MCUs: to decode ~90 second
> dits:
> https://www.qsl.net/m0ayf/What-is-QRSS.html :
> "QRSS generally means that the CW sending speed is below 2-3 WPM - usually
> much slower than that. Let's take as a rather extreme example, the VA3LK
> beacon on 137.79 Hz. This experimental beacon has operated at a "dit" rate
> of one dit every ninety seconds - that's about .0133 words-per-minute, or
> about 0.8 words-per-hour. This also implies that the detection bandwidth
> for such a signal (see the equations below
> <http://www.ka7oei.com/qrss1.html#bw_speed>) should be at least 0.033 Hz
> - that's 33 Millihertz (mHz - notice the small "m"!) Going much much
> narrower than this and the "dits" will start to run together and sacrifice
> "intelligibility.
>
> Compare this to the 2.4 Hz bandwidth for a moment: 33 mHz (notice the
> small "m") is 72000 times narrower than an SSB bandwidth - a difference of
> over 48 db. A better example would be to compare this with a 30 Hz CW
> filter - a "reasonable" value for a well-designed CW filter. This would
> still be 900 times narrower than that - over 29 db difference in the
> receive signal/noise ratio. (Once again, we are ignoring psychoacoustics.)"
>
> Then integrating on a Single board computer, a batteryless-centered
> design- i.e supercaps or lithium ion capacitors:
> https://www.youtube.com/watch?v=LDzxifG7VeA&t=237s <-(similar to the
> batteryless bluetooth beacon, but on ham bands)
>
> https://www.tindie.com/products/jaspersikken/solar-harvesting-into-lithium-ion-capacitor/
> :
> "If your application draws 30mA from the 3.3V output, then it will run
> for: 250F(3.78V-2.49V)90%/0.03A= 10750s = 2.7 hrs"
>
> In the U.S., the maximum transmit power for MedFER with a 3 meter antenna
> is 100mW:
> https://lwca.net/sitepage/part15/index.htm
> http://www.ka7oei.com/psk_medfer.html
> A portable phone wouldn't be able to use 3 meter antenna- maybe only 7
> centimeters or 30cm thus lower range would be expected, but the idea here
> is to explore bands other than LoRa to transmit even a few characters all
> on solar power (and initially, without GPS, since today's tech probably
> can't run it without larger solar panels in real time).
>
> So theoretically, one could have a portable, fully solar powered phone
> transmitting a signal for 2.7hrs running at 100mW to send a few bytes of
> data (the microcontroller alone uses less than 1mA: 5uAx 96Mhz=960uA)
>
> While I understand rechargeable batteries are not super expensive, the
> idea is that by using lithium ion capacitors, or supercaps, it would create
> a positive-feedback loop of developing slower transmission methods for
> lower priority projects (as an amateur project). Thanks for reading!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/markqvist/Reticulum/discussions/171>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/APRMG4TASHR3TMUDERD6OQ3WLOBJ5ANCNFSM6AAAAAASS3SQN4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Pardon the brevity, sent from a mobile device. So there.

**Comment by hatonthecat** [comment]
"Could even do a digital signage type application. Doesn’t matter if it
takes a few try’s to get the data through .

Steve  N0FPF"

Yes, there could be many applications for this- in fact, e-paper displays use less overall power per day if they have use a limited number of refreshes per day: https://rdotdisplays.com/articles/resources/rdot_eink_reflective_lcd.jpg
